drivers/ethernet/Kconfig.renesas_ra

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config ETH_RENESAS_RA
+	bool "Renesas RA Ethernet"
+	default y
```

Comment:
This change to `Kconfig.renesas_ra` looks like it is in wrong commit, it should be in Ethernet driver addition commit instead.

Reply:
Thank you for reviewing, I moved this to correct commit